src/cljs/rems/administration/create_form.cljs

```diff
@@ -87,13 +87,20 @@
   (and (not (str/blank? (:key option)))
        (valid-required-localized-string? (:label option) languages)))
 
+(defn- parse-maxlength [maxlength]
+  (let [parsed (parseInt maxlength)]
+    (when-not (js/isNaN parsed) parsed)))
+
 (defn- valid-request-field? [field languages]
   (and (valid-required-localized-string? (:title field) languages)
        (boolean? (:optional field))
        (not (str/blank? (:type field)))
        (if (supports-input-prompt? field)
          (valid-optional-localized-string? (:input-prompt field) languages)
          (nil? (:input-prompt field)))
+       (if (supports-maxlength? field)
+         (not (neg? (:maxlength field)))
+	 (nil? (:maxlength field)))
```

Comment:
You should use only spaces for indentation. The indentation on this line was wonky.